CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
+The format conforms to [keepachangelog.com](http://keepachangelog.com/).
```

Comment:
I would have the link title be "Keep a Changelog", reflecting the title on the website. Also love the idea!